app/api/resources/user.py

```diff
@@ -60,9 +60,10 @@ def get(cls):
 
         page = request.args.get("page", default=UserDAO.DEFAULT_PAGE, type=int)
         per_page = request.args.get("per_page", default=UserDAO.DEFAULT_USERS_PER_PAGE, type=int)
+        is_verified = request.args.get("is_verified") == 'true'
```

Comment:
What happens if the `is_verified` field does not come with the query (i.e.: is None)? I think the way it is now, it will be false, and then `list_users()` will return non verified users. What if I want to query every user (verified and non-verified)?